src/components/ContentEditable.native.tsx

```diff
@@ -43,18 +44,33 @@ const ContentEditable = ({
   forceUpdate,
   placeholder,
   isEditing,
+  isTable,
   ...props
 }: ContentEditableProps) => {
   const allowInnerHTMLChange = useRef<boolean>(true)
   const { width } = useDimensions().window
   const [height, setHeight] = useState(DEFAULT_WEBVIEW_HEIGHT)
   const contentRef = useRef<WebView>(null)
+  const [innerHTMLValue, setInnerHTMLValue] = useState(html)
```

Comment:
Seems like `innerHTMLValue` doesn't need to be a reactive state. If so, you can use `mutable` ref instead.